### PR TITLE
add Uptime and support translations

### PR DESCRIPTION
**Pull request by slametps**
add Uptime and support translations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ git clone https://github.com/BenRoe/MMM-SystemStats
 
 Navigate to the new `MMM-SystemStats` folder and install the node dependencies.
 ```
-cd MMM-SystemStats/ && npm install
```
Comment (BenRoe) [inline README.md]: @slametps why did you removed the `cd MMM-SystemStats`? It's easier for noobs.

File: README.md [hunk 2]
```diff
@@ -48,7 +48,6 @@ modules: [
 		config: {
 			updateInterval: 10000,
 			animationSpeed: 0,
-			align: 'right', // align labels
```
Comment (BenRoe) [inline README.md]: @slametps  Removed unintended?

File: README.md [hunk 3]
```diff
@@ -86,10 +85,9 @@ The following properties can be configured:
 		</tr>
 
 		<tr>
-			<td><code>align</code></td>
```
Comment (BenRoe) [inline README.md]: Same here.

File: node_helper.js [hunk 4]
```diff
@@ -10,6 +10,7 @@
 const NodeHelper = require('node_helper');
 var async = require('async');
 var exec = require('child_process').exec;
+var request = require('request');
```
Comment (BenRoe) [inline node_helper.js:13]: Is the request dependency installed with MM or is it necessary to install it via npm install?
If via npm please add it to the `package.json` file.

File: README.md (hunk 1 above)
Comment (slametps) [reply README.md]: in my repo, I don't remove that line. I wonder myself, why github assumed it is removed. I'm retrying re-submit my repo.

File: README.md (hunk 2 above)
Comment (slametps) [reply README.md]: in my repo, I don't remove that line. I wonder myself, why github assumed it is removed. I'm retrying re-submit my repo.

File: README.md (hunk 3 above)
Comment (slametps) [reply README.md]: in my repo, I don't remove that line. I wonder myself, why github assumed it is removed. I'm retrying re-submit my repo.

**Review by slametps — COMMENTED** [review]
in my repo, I don't remove that line. I wonder myself, why github assumed it is removed. I'm retrying re-submit my repo.

**Review by BenRoe — APPROVED** [review]

**Pull request merged**